### ISSUE TITLE
Final Report Peer Review

### ISSUE THREAD

**Issue by alutterb** [kelly022/ORIE-4741-FALL-2020-PROJECT#13]
This project aims to predict car price listing using features such as make, model, trim, year, accidents, etc. Overall the report had no logical inconsistencies and the methods used seemed appropriate in each context. Things I particularly like about the project include choosing good loss functions as well as regularizers that mesh well with the nature of the problem. They initially started out with many features that may cause overfitting, but effectively cut it down by more than half to a more manageable 34 features. I also like how they collected their data by their own means as well as used GLRMS to impute any holes in the data. I also thought the discussion of fairness and WMD to be sufficient.

Some of the issues I had were that the features were somewhat confusing with a lot of acronyms being used. The boxplots were not labelled although the figure caption helps with this. I would have also like a bit more illumination on the process of feature reduction. One main concern is what value does this project add over current services? What is unique to your project that cannot be captured by the KBB system?

Overall the project follows the script well for data preprocessing -> model building -> analysis, but I would like to have seen more discussion on differentiation from other types of models that are readily accessible.
